Configuration/AlCa/python/autoCondPhase2.py

```diff
@@ -12,61 +12,40 @@
 ##
 ## Active geometries: https://github.com/cms-sw/cmssw/blob/master/Configuration/Geometry/README.md
 ##
-## T15: Phase2 tilted tracker (v6.1.6) w/ phase 2 pixel (v6.1.3) (Active geometry: same as T14. Material Budget: major update in IT, gathering info from recent Mechanical designs.)
 ## T21: Phase2 tilted tracker. Outer Tracker (v8.0.0): TBPS update in Layer 1 (facilitate IT insertion) + In all TEDD, update sensors Z inter-spacing. Inner Tracker: (v6.1.5) from previous T17
 ## (TFPX: Changed sensors spacing within all double-disks + Increased distance between Disks 6 and 7 + TBPX portcards between Disks 6 and 7.)
-## T25: Phase2 tilted tracker. Outer Tracker (v8.0.0): same as T24/T21. Inner Tracker (v7.0.2): Based on (v6.1.5) (T24/T21), but with 3D sensors in TBPX L1.
-## T26: Phase2 tilted tracker. Outer Tracker (v8.0.0): same as T24/T21. Inner Tracker (v7.0.3): Based on (v6.1.5) (T24/T21), but with 3D sensors in TBPX L1 and 50x50 pixel aspect ratio in TFPX and TEPX.
```

Comment:
@emiglior can you please confirm that there is no need to maintain T25 / T26 conditions anymore at this point?

Reply:
T25 and T26 are still active geometries.
We can probably dismiss T26 (square pixels in the Inner Tracker discs) but we must keep T25 as this is the only geometry with 3D pixels in TBPX L1 (actually we can start to produce regularly RelVals based on this geometry).

Reply:
IIUC T25 was contained in `D80 = T25+C14+M9+I13+O7+F6` and that geometry was removed in PR https://github.com/cms-sw/cmssw/pull/38702 as a follow-up of issue https://github.com/cms-sw/cmssw/issues/38594, so it won't be possible to submit any relvals with it, unless it's brought back. @srimanob

Reply:
Does it fit with D88, D92, i.e. with current MTD ?

Reply:
> IIUC T25 was contained in `D80 = T25+C14+M9+I13+O7+F6` and that geometry was removed in PR #38702 as a follow-up of issue #38594, so it won't be possible to submit any relvals with it, unless it's brought back. @srimanob

It will be difficult to bring it back as HGCAL are working on cleaning, and would like to keep only V16 and V17 of HGCAL (C17 or later). If my above comment is good, we can go for it.

Reply:
@srimanob, I don't think Tracker particularly cares about what other subsytem versions are used to build the geometry, provided that there is at least one geometry containing T25 (or T26, though from the comment above https://github.com/cms-sw/cmssw/pull/39571#discussion_r985488927 it's not clear to me if it's still useful or not...)

Reply:
Any other combinations of the other detectors is ok for the Tracker provided we can produce RelVals with T25.
What about having a geometry derived from D86 but featuring T25 instead of T24, e.g.
Dxx = T25+C17+M10+I14+O8+F6

Reply:
Hi @emiglior 
If there is no constraint on version of MTD, please go with DXX = D88+T25 because D88 is the current Phase-2 baseline now.

Reply:
Fine with me! Who is going to prepare the PR?

Reply:
I will do it.